requirements.txt

```diff
@@ -3,4 +3,7 @@ boost@1.79 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build -DCMAKE_CXX_FLAGS=" -std
 half,https://github.com/pfultz2/half/archive/1.12.0.tar.gz -X header -H sha256:0a08660b68abb176ebc2a0cdf8de46e3182a7f46c66443bb80dbfaaec98cf969 --build
 ROCmSoftwarePlatform/rocMLIR@rocm-5.5.0 -H sha256:a5f62769d28a73e60bc8d61022820f050e97c977c8f6f6275488db31512e1f42 -DBUILD_FAT_LIBROCKCOMPILER=1
 nlohmann/json@v3.9.1 -DJSON_MultipleHeaders=ON -DJSON_BuildTests=Off
+ROCmSoftwarePlatform/FunctionalPlus@v0.2.18-p0
+ROCmSoftwarePlatform/eigen@3.4.0
+ROCmSoftwarePlatform/frugally-deep@9683d557eb672ee2304f80f6682c51242d748a50
```

Comment:
If these dependencies are not installed, then CMake issues the following error message:
```
Adding Test: solver_convasm3x3u
-- Configuring done
CMake Error in src/CMakeLists.txt:
  Found relative path while evaluating include directories of "MIOpen":

    "FDEEP_INCLUDE_DIR-NOTFOUND"

<many similar lines>

CMake Error in test/gtest/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "test_mt_queue":

    "FDEEP_INCLUDE_DIR-NOTFOUND"



-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```
This should be replaced by some more clean error message.

Reply:
Added descriptive FATAL_ERROR message if either frugally-deep or eigen not found.

Reply:
These are WARNINGs now, please fix.

Reply:
I switched them to warnings based upon a meeting with Jehandad. The warning also makes sense to me since I set MIOPEN_ENABLE_AI_KERNEL_TUNING to False if these dependencies are not installed.

Reply:
I will add to the message that I am setting MIOPEN_ENABLE_AI_KERNEL_TUNING off.

Reply:
@Dmantri98 
>>> Added descriptive FATAL_ERROR message if either frugally-deep or eigen not found.
>>
>> These are WARNINGs still, please fix.
>
> I switched them to warnings based upon a meeting with Jehandad.

No problem, but in the future -- please try to timely provide valid information in your comments. That would save reviewers' time and efforts. Thanks in advance.

@Dmantri98 @JehandadKhan 

Not resolved yet. If I provide `-DMIOPEN_ENABLE_AI_KERNEL_TUNING=On` and some of the required deps not found, then CMake should fail (instead of issuing a warning and resetting MIOPEN_ENABLE_AI_KERNEL_TUNING). This is important for automated testing (and that is why this is our current practice).

Reply:
@Dmantri98 Please unresolve this thread, thanks.

Reply:
Resolved.